tpm/errors.go

```diff
@@ -0,0 +1,9 @@
+package tpm
+
+import "errors"
+
+// ErrNotFound is returned when a Key or AK cannot be found
+var ErrNotFound = errors.New("not found")
```

Comment:
maybe prefix these errors with `tpm: ` like, `tpm: not found`.

Reply:
These errors are wrapped internally. The resulting error would be something like `failed getting AK "name": not found`. Prefixing these two errors won't look nice. But I could prefix the top level errors returned from the `tpm` package.